### PR TITLE
api reference documentation added

### PR DESCRIPTION
**Pull request by DanielAugustoUruena**
## Summary

- API documentation added

## Known Issues

None.

## Trello Card

[Documentación de Sign Up](https://trello.com/c/BiITdd5i/30-documentaci%C3%B3n-de-sign-up)

### CODE REVIEW

File: apiReference.json [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+          "400": {
+            "description": "Missing parameters",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                },
+                "example": {
+                  "message": "the field 'email' is required",
+                  "internal_code": "input_data_error"
```
Comment (jonnyalexbh) [inline apiReference.json]: do you think the 500 & 503 error would be missing?

File: apiReference.json [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        "type": "integer",
+        "description": "Users identifier",
+        "example": 1
+      },
+      "firstName": {
+        "type": "string",
+        "description": "First name of user",
+        "example": "Jhon"
+      },
+      "lastName": {
```
Comment (jonnyalexbh) [inline apiReference.json]: Good camel_Case 🥇

File: apiReference.json [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    }
+  },
+  "components": {
+    "schemas": {
+      "id": {
+        "type": "integer",
+        "description": "Users identifier",
+        "example": 1
+      },
+      "firstName": {
```
Comment (jonnyalexbh) [inline apiReference.json]: I think you can define which fields are required in the body

File: apiReference.json [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+          }
+        }
+      },
+      "UserResponse": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "$ref": "#/components/schemas/id"
+          },
+          "first_name": {
```
Comment (jonnyalexbh) [inline apiReference.json]: Good Snake Case

File: apiReference.json (hunk 1 above)
Comment (DanielAugustoUruena) [reply apiReference.json]: errors documentation added

File: apiReference.json (hunk 3 above)
Comment (DanielAugustoUruena) [reply apiReference.json]: fields marked as required, although this mark is not visible.

**Review by jxczz2 — APPROVED** [review]

**Review by jonnyalexbh — APPROVED** [review]
great!

**Pull request merged**